scripts/dump_data.py

```diff
@@ -0,0 +1,151 @@
+import click
+from pathlib import Path
+from shutil import copy, make_archive
+import pandas as pd
+
+import mmeds.config as fig
+
+"""
+Script that collects data about a study or studies, and their associated sequencing runs and analysis configs,
+    and converts it into a ZIP archive that can be read back into MMEDS using load_data.py
+"""
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('-i', '--input-study-names', required=True, help='Text file with list of study directories to include')
+@click.option('-p', '--path', required=False, default='.', help='Base path that includes study directories')
+@click.option('-o', '--output-zip', required=False, default='mmeds_dump', help='Name for output zip file')
+@click.option('-u', '--user', required=False, default='adamcantor22', help='Account that owns the used runs')
+@click.option('-nz', '--no-zip', required=False, is_flag=True, help='Whether or not to zip the final structure')
+def dump(input_study_names, path, output_zip, user, no_zip):
```

Comment:
docstring
this function is doing too many things, requiring multiple returns
break up dump into a couple functions that are imported from mmeds.util or wherever they fit best